File: Loja - C++/Usuario.cpp
#include "Usuario.h"

using namespace std;

void Usuario::setUsuario(string name, string key, string cpf)
{
    setNome(name);
    setSenha(key);
    setCpf(cpf);
}

void Usuario::setNome (string name)
{
        nome = name;
}

void Usuario::setSenha (string key)
{
        senha = key;
}

void Usuario::setCpf (string cpf)
{
        this->cpf = cpf;
}

string Usuario::getNome()
{
    return(nome);
}

string Usuario::getSenha()
{
    return(senha);
}

string Usuario::getCpf()
{
    return(cpf);
}

int Usuario::gerenciaEstoque(string name)
{
    int op;
    cout << "******** Bem vindo ao controle de estoque da loja ********" << endl;
    cout << "Usuario logado: " << name << endl;
    cout << "Informe a opcao desejada: " << endl;
    cout << "1 - Inserir novos produtos" << endl;
    cout << "2 - Retirar produtos do estoque" << endl;
    cout << "3 - Checar estoque" << endl;
    cout << "4 - Desfazer conta" << endl;
    cout << "0 - Sair" << endl;
    cin >> op;
    return op;
}

File: Loja - Java/src/sporthub/Usuario.java
package sporthub;
import java.util.Scanner;
/**
 *Concluido
 * @author Weslles
 */
public class Usuario {
    private String nome;
    private String senha;
    private String cpf;
    
    public void setUsuario( String name, String key, String cpf ){
        setNome(name);
        setSenha(key);
        setCpf(cpf);
    }
    public void setNome(String name){
        nome = name;
    }
    public void setSenha(String key){
        senha = key;
    }
    public void setCpf( String cpf){
        this.cpf = cpf;
    }
    public String getNome(){
        return nome;
    }
    public String getCpf(){
        return cpf;
    }
    public String getSenha(){
        return senha;
    }
    public int gerenciaEstoque(String name){
        int opcao;
        Scanner ler = new Scanner(System.in);
        
        System.out.println("\n******** Bem vindo ao controle de estoque da loja ********\n");
        System.out.println("Usuario logado "+name);
        System.out.println("\nInforme a opção desejada: ");
        System.out.println("1 - Inserir novos produtos");
        System.out.println("2 - Retirar produtos do estoque");
        System.out.println("3 - Checar estoque");
        System.out.println("4 - Desfazer conta");
        System.out.println("0 - Sair");
                
        opcao = ler.nextInt();
        return opcao;
    }
}

